src/rules/unused_record_fields.erl

```diff
@@ -131,7 +131,9 @@ find_record_field(FieldName, Definitions) ->
 
 %% @todo Add ignore pattern support
 -spec ignored(hank_rule:ignore_pattern(), term()) -> boolean().
-ignored(undefined, _IgnoreSpec) ->
-    false; %% Remove this clause and just use the one below
-ignored(_Pattern, _IgnoreSpec) ->
-    true.
+ignored({RecordName, FieldName}, {RecordName, FieldName}) ->
+    true;
+ignored({RecordName, _FieldName}, RecordName) ->
```

Comment:
I think this case is not tested

Reply:
It is. With `ignored_record`.

Reply:
Here: https://github.com/AdRoll/rebar3_hank/pull/104/files#diff-e7a3faa393edcba8b801bc08843eade4181562b7d76ebfa841e5ef70921fdc81R6

Reply:
Yes, for this clause:
```erlang
ignored({RecordName, FieldName}, {RecordName, FieldName}) ->
```
but for this:
```erlang
ignored({RecordName, _FieldName}, RecordName) ->
```
I think it's missing something like:
```
-hank([{unused_record_fields,
          [ignored_record,
           {a_record, ignored_field_1},
           {a_record, ignored_field_2},
           {another_complete_record}
           ]}]).
```

Reply:
The whole `ignored_record` is ignored as Brujo pointed out, so it's covered.

Reply:
Yes, sorry, I got confused between the rulename and the recordname